src/System.Time/src/System/Date.cs

```diff
@@ -0,0 +1,1629 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace System
```

Comment:
We can decide later on this (i.e. don't block the initial commit), but I wonder if System.Time would not be a better namespace. One thing we want to avoid is confusing existing developers who are happy with DateTimeOffset.